src/controllers/routineController.js

```diff
@@ -0,0 +1,18 @@
+const { routineStartService } = require("../services");
+
+const routineStartController = async (req, res) => {
+  try {
+    const { routineId } = req.params;
+    
+    const routineStart = await routineStartService(routineId);
```

Comment:
```suggestion
    const routineStart = await getExerciseRoutineByRoutineId(routineId);
```
마찬가지입니다.
- 뒤에 `controller`, `service` 붙이는건 팀내 합의에 따라 주세요 :)